all-paths-from-source-to-target.py

```diff
@@ -1,18 +1,18 @@
-class Solution:
-    def allPathsSourceTarget(self, graph: List[List[int]]) -> List[List[int]]:
-        paths = []
-        n = len(graph)
```

Comment:
Liked the fact you store len(graph) since you will use is repetitively in you code